packages/donor/src/components/AppHeader/AppHeader.tsx

```diff
@@ -49,7 +49,7 @@ export default function AppHeader({
   if (hasBurgerMenu) {
     icon = (
       <IconButton
-        buttonName="burger_menu"
+        analyticsName="burger_menu"
```

Comment:
Why is this renaming part of this PR?

Please split this PR to a refactor one and a UI one

Reply:
Done